docs/guides/data_documentation.rst

```diff
@@ -105,12 +108,20 @@ Here is an example of a site configuration:
                 module: great_expectations.render.view
                 class: DefaultJinjaPageView
 
+* ``validations_store`` and ``profiling_store`` in the example above specify the location of validation and profiling results that the site will include in the documentation. The store's ``type`` can be ``filesystem`` or ``s3``. S3 store is not currently implemented, but will be supported in a near future. ``base_directory`` must be specified for ``filesystem`` stores. The optional ``run_id_filter`` attribute allows to include (``eq`` for exact match) or exclude (``ne``) validation results with a particular run id.
```

Comment:
S3 store is not currently implemented, but will be supported in a near future.
->
(S3 store is not currently implemented, but will be supported in the near future.)

Reply:
implemented

Reply:
implemented